schema.graphql

```diff
@@ -32,7 +44,8 @@ type Transaction @entity {
   gasWanted: BigInt!
   fees: [Coin]!
   memo: String
-  status: TxStatus!
+  # 0 = Success, 1 = Error
+  status: Int!
```

Comment:
Why replace this enum type?

Reply:
For reference:

![image](https://github.com/user-attachments/assets/29c6343c-534e-41ad-bbe3-e6ef136df960)

Reply:
Currently we can't use Enums in the `schema.graphql` as we will need to query by those fields, so for the moment we will keep the Int type. [See here why we can't do it with Enums.](https://github.com/pokt-network/pocketdex/issues/27)